mphx/client/EventManager.hx

```diff
@@ -11,17 +11,44 @@ class EventManager
 		eventMap = new Map();
 	}
 
+	/**
+	 * Add a event.
+	 * If the event already exist, override the precedent callback
+	 * @param	eventName
+	 * @param	event
+	 */
 	public function on(eventName:String, event:EventFunction)
 	{
-		eventMap.set(eventName,event);
+		eventMap.set(eventName, event);
+	}
+	
+	/**
+	 * Remove an event if exist
+	 * @param	eventName
+	 */
+	public function remove(eventName:String)
+	{
+		if (eventMap.exists(eventName)) 
+		{
+			eventMap.set(eventName,null);
+			eventMap.remove(eventName);
+		}
 	}
 
+	/**
+	 * call an event if exist.
+	 * If the callback associed to the event is null. nothing append
+	 * @param	eventName
+	 * @param	data
+	 */
 	public function callEvent(eventName:String, data:Dynamic)
 	{
 		//If an event with that name exists.
-		if (eventMap.exists(eventName)) {
+		if (eventMap.exists(eventName)) 
+		{
 			//See if the event should be called with or without the sender.
-			eventMap.get(eventName)(data);
+			if(eventMap.get(eventName) != null)
```

Comment:
Hum, depends if you allow a null callback... Might be usefull for desactive temporary an event without destroy it. 
About throwing an error, i'm not sure that it's necessary. In my mind, it's not an important error. Maybe return true/false if the add had work ? I prefer let @5Mixer decide about this point.